charts/argocd-apps/values-test.yaml

```
@@ -38,24 +179,78 @@ applications:
     replicaCount: 1
 - name: frontend
   helmValues:
-    image:
+    appImage:
       repository: 172025368201.dkr.ecr.eu-west-1.amazonaws.com/frontend
       tag: latest  # To be edited by automation (not yet implemented).
     replicaCount: 1
-    appMemcacheServers:
-    - frontend-memcached.eks.test.govuk-internal.digital
+    extraEnv:
+      - name: SECRET_KEY_BASE
+        value: secret
```

I'd stick a TODO on this (and any other occurrences of it) just to make it a bit easier to find.
```suggestion
        value: secret  # TODO: set SECRET_KEY_BASE and make it a secret
```